nixos/modules/services/networking/bind.nix

```diff
@@ -103,36 +103,40 @@ in
       };
 
       listenOn = mkOption {
-        default = ["any"];
+        default = [ "any" ];
         type = types.listOf types.str;
         description = "
           Interfaces to listen on.
         ";
       };
 
       listenOnIpv6 = mkOption {
-        default = ["any"];
+        default = [ "any" ];
         type = types.listOf types.str;
         description = "
           Ipv6 interfaces to listen on.
         ";
       };
 
       zones = mkOption {
-        default = [];
+        default = [ ];
         description = "
           List of zones we claim authority over.
             master=false means slave server; slaves means addresses
            who may request zone transfer.
         ";
-        example = [{
-          name = "example.com";
-          master = false;
-          file = "/var/dns/example.com";
-          masters = ["192.168.0.1"];
-          slaves = [];
-          extraConfig = "";
-        }];
+        example = literalExample ''
```

Comment:
Why this change?

Reply:
@Ma27 requested it:
https://github.com/NixOS/nixpkgs/pull/95050#discussion_r467759793

Reply:
@alyssais `literalExample` results in better formatting in the manual.